interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/CarecoordinationTable.php

```diff
@@ -22,6 +22,7 @@
 use Laminas\Db\TableGateway\AbstractTableGateway;
 use OpenEMR\Services\Cda\CdaTemplateImportDispose;
 use OpenEMR\Services\Cda\CdaTemplateParse;
+use OpenEMR\Services\Cda\CdaValidateDocuments;
```

Comment:
@adunsulag Note I've made changes if you're in this class

Reply:
I haven't touched that class, but thanks for giving me the heads up.